02-text-files.Rmd

```diff
@@ -284,23 +284,27 @@ commands, in particular.
 
 To run vi, type:
 ```
-vim 2cities/book.txt
+vi 2cities/book.txt
 ```
-You should see all your changes from before, right?
+(You should see all your changes from before, right?)
 
-To exit, type:
+vi starts in "normal mode", which allows you to navigate around the file.
+In normal mode, what you type does not change the file - instead, it lets
+you issue commands to vim.
+
+The first and most important (?) command - to exit, type:
 ```
 :q
 ```
 and if there are no changes, it will simply exit.
 
 Run vi again, and let's edit --
 ```
-vim 2cities/book.txt
+vi 2cities/book.txt
 ```
 and then type 'i' to go to "insert" mode, and type something.
 
-Then hit the escape key to go back to "view" mode.
+Then hit the escape key to go back to "normal" mode.
```

Comment:
there are a few more places in this section that use the "view mode" terminology - should it be switched to "normal mode" too?

Reply:
fixed, thanks!